package.json

```diff
@@ -5,8 +5,10 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack -p",
-    "cypress:open": "cypress open",
-    "cypress:run": "cypress run",
+    "cypress:open:localhost": "cypress open",
```

Comment:
We created more scripts based on environment in our apps like cypress:open:testing, staging, etc. to account for different base URLs for different environments. We would also set environment variables so our tests can load up a different object of data to use specifically for that environment i.e. different api keys, credentials, other env specific/user specific information. Yours won't get that complicated but to give ya insight into how we expanded this 😄